pcre/CMakeLists.txt

```diff
@@ -125,7 +125,7 @@ OPTION(PCRE_BUILD_PCRE16 "Build 16 bit PCRE library" OFF)
 
 OPTION(PCRE_BUILD_PCRE32 "Build 32 bit PCRE library" OFF)
 
-OPTION(PCRE_BUILD_PCRECPP "Build the PCRE C++ library (pcrecpp)." ON)
+OPTION(PCRE_BUILD_PCRECPP "Build the PCRE C++ library (pcrecpp)." OFF)
```

Comment:
Literal unmodified source dump other than the previous commit (1b3856f6407bfdcfdef342cd3d0d830f57cc3704) where you modified that same file?